manifest.php

```diff
@@ -29,7 +29,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '17.17.0',
+    'version'     => '17.17.1',
```

Comment:
As there is a dependency on other extension, pease also update the version in the `requires` clause below.

Reply:
Ok, thx, I'll changes this in taoQtiItem repo.

Reply:
Well, the change is not meant to be made in the other extension, but in this one. In the `requires` clause there is the required version of `taoQtiItem`. You should increase this number in this file to ensure the requirements are met.

Reply:
Hmmm, as I understand it will look like this:
We are changing this repo version. We are saying that this repo requires at least __some__ version of the other repo. But it is not this repo relying on other repo changes but these repo changes are need for other repo to work. Maybe it's better to change `taoQtiItem` repo `requires` block? As it needs at least __some__ version of this repo? Am I wrong? Or it doesn't matter?
p.s.
Sorry for having so much questions :)

Reply:
No problem for the questions. We are always learning, and the better way is to exchange info.

So, you should see the requirements as a hierarchical tree. `taoQtiItem` only takes care of items. `taoQtiTest`, takes care of tests, that contain items. So `taoQtiTest` is using `taoQtiItem`, and it requires it. In the other hand, `taoQtiItem` can be used outside of `taoQtiTest`, and don't require it. See that as boxes containing boxes. `taoQtiItem` cannot even know about `taoQtiTest`, otherwise you will go into impossible situations, where each one is the father of the other.
For some features, `taoQtiTest` must ensure the other extensions it is relying on are in the right version that bring the wanted behavior/api. So it must declare the minimum version of the needed dependencies.

Is that clear enough? :)

Reply:
Yeah, thanks, now, I think, I understood what you wanted me to changed and, the main part, - why :)